From 83f01e877fb6b77692c66cbd3f1c6c95217b721d Mon Sep 17 00:00:00 2001
From: Henrik Riomar <henrik.riomar@gmail.com>
Date: Wed, 13 Mar 2019 09:32:40 +0100
Subject: [PATCH] zfs-import: should be before swap

zfs-import must be done before swap in order for swap on zvol to work
---
 etc/init.d/zfs-import.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/init.d/zfs-import.in b/etc/init.d/zfs-import.in
index fea661bbc..420d2e8a7 100644
--- a/etc/init.d/zfs-import.in
+++ b/etc/init.d/zfs-import.in
@@ -38,6 +38,7 @@
 
 do_depend()
 {
+	before swap
 	after sysfs udev
 	keyword -lxc -openvz -prefix -vserver
 }
-- 
2.21.0

